extensions/ql-vscode/src/model-editor/model-editor-view.ts

```diff
@@ -655,11 +655,14 @@ export class ModelEditorView extends AbstractWebview<
       this.databaseItem,
       methodSignatures,
     );
+    const processedByAutoModelMethods =
+      this.modelingStore.getProcessedByAutoModelMethods(this.databaseItem);
```

Comment:
Oops, I implemented the `methodSignatures` argument for this method but then forgot to use it here.

I've added that in now and done another manual test that everything is working ok.